mungegithub/mungers/blunderbuss.go

```diff
@@ -125,11 +126,25 @@ func getPotentialOwners(author string, feats *features.Features, files []*github
 			if owner == author {
 				continue
 			}
+			// Inside of a vendor directory, it is possible that
```

Comment:
This isn't specific to vendor. We've had people leave the org and run into this problem. Or typo'd entries in OWNERS. So I don't think this should be restricted to vendor at all.